r/vignettes/python.Rmd

```diff
@@ -7,9 +7,13 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-The `arrow` package provides `reticulate` methods for passing data between
+The arrow package provides [reticulate](https://rstudio.github.io/reticulate/) methods for passing data between
 R and Python in the same process. This document provides a brief overview.
 
+Why you might want to use `pyarrow`?
+* To use some Python functionality that is not yet implemented in R, for example, the `concat_arrays` function.
+* To transfer Python objects into R, for example, a Pandas dataframe into an R Arrow Array. 
```

Comment:
```suggestion
Why you might want to use `pyarrow`?

* To use some Python functionality that is not yet implemented in R, for example, the `concat_arrays` function.
* To transfer Python objects into R, for example, a Pandas dataframe into an R Arrow Array. 
```
Needs a space to render properly

Reply:
Ok, I see I need to build the docs next time, sorry for giving you additional work! I wanted to try how it goes ...